static/src/js/utils/getRevisionCount.js

```diff
@@ -0,0 +1,20 @@
+import { capitalize } from 'lodash-es'
+import useRevisionsQuery from '../hooks/useRevisionsQuery'
+
+/**
+ * Given a formData with a an array, getRevisionCount will return the length
+ * of of the items
+ * @param {String} conceptId The conceptId we use to query revision Count
+ */
+const getRevisionCount = (conceptId, type) => {
```

Comment:
This should not be a utility function, just use the hook we created in the one place this utility is used.